tools/tf_env_collect.sh

```diff
@@ -19,6 +19,7 @@ set -u  # Check for undefined variables
 echo "Collecting system information..."
 
 OUTPUT_FILE=tf_env.txt
+python_bin_path=$(which python || which python3 || true)
```

Comment:
Good point, had just taken from [`configure#L76`](https://github.com/tensorflow/tensorflow/blob/master/configure#L76)
So, should we display an appropriate error message?